app/code/core/Zend/Validate/EmailAddress.php

```diff
@@ -324,7 +324,7 @@ public function setDeepMxCheck($deep)
     /**
      * Returns the set domainCheck option
      *
-     * @return unknown
+     * @return string
```

Comment:
See line 343: `$this->_options['domain'] = (boolean) $domain;`

```suggestion
     * @return bool
```